spec/predicates/README.md

```diff
@@ -12,17 +12,20 @@ community! Please see our [New Predicate Guidelines].
 This directory contains predicate specification types that have gone through
 our [vetting process], and may be of general interest:
 
--   [SLSA Provenance]: To describe the origins of a software artifact.
+-   [SLSA Provenance]: To describe the build process of a software artifact.
```

Comment:
Not sure I agree with this change.  The provenance does also describe the source location. Also, the definition of provenance includes 'origin'.

Alternatively perhaps we can reword [what SLSA says about it](https://slsa.dev/provenance/v1-rc1#:~:text=Describe%20how%20an%20artifact%20or%20set%20of%20artifacts%20was%20produced%20so%20that%3A) -> "Describes how an artifact or set of artifacts was produced"